### DISCUSSION TITLE
Listed actions demands login for every post

### DISCUSSION THREAD

**Discussion by WinstonSmith73** [standardnotes/forum#2939] [Suggestions]
**Describe the bug**
Every time I go to publish a note to my Listed.to blog I am asked to re-enter my Standard Notes password, even though I select "1 week."

**To Reproduce**
1. Go to Listed actions
2. Enter SN password, selecting any time period
3. Go to Listed actions, Publish to Blog
4. Select next note to publish
5. Go to Listed actions, ASKS FOR PASSWORD AGAIN

**Expected behavior**
I expect the password for publishing to my Listed.to blog to stick for the specified amount of time, not have to re-enter it for every post.

**Screenshots**
![Listed Publish  - 1](https://user-images.githubusercontent.com/101750466/213306398-e7009e70-a0a9-4354-a84a-c12431526f97.jpeg)
![Listed Publish  - 2](https://user-images.githubusercontent.com/101750466/213306469-f7213713-f422-4021-a800-1e5cba620515.jpeg)

**Desktop (please complete the following information):**
 - OS: macOS 12.6.2
 - Browser: NA [using the macOS app]
 - Desktop App Version 3.139.0

Edit to Add: same behavior in the web app in both Brave and Firefox (current versions).

**Comment by effieeee** [comment]
If I recall correctly, the team was keeping it in consideration to change the authentication behavior and requirements for Listed posts 👍

**Comment by WinstonSmith73** [comment]
I really hope they change it- it is ridiculously tedious as-is.
